components/create_comment/create_comment.jsx

```diff
@@ -661,7 +661,7 @@ class CreateComment extends React.PureComponent {
         }
     }
 
-    handleUploadError = (err, clientId = -1, rootId = -1) => {
+    handleUploadError = (err, clientId = -1, currentChannelId, rootId = -1) => {
```

Comment:
maybe setting the default value to `-1` to be consistent with the rest of the other parameters?

Reply:
Lint complains since the value is never used in this function.